ui/packages/consul-ui/app/components/certificate/index.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class TopologyMetrics extends Component {
```

Comment:
If you don't want to go with the `set` helper the name of this class should be `Certificate`.